gstools/field/base.py

```diff
@@ -384,59 +320,62 @@ def model(self, model):
                 "Field: 'model' is not an instance of 'gstools.CovModel'"
             )
 
+    @property
+    def mean(self):
+        """:class:`float` or :any:`callable`: The mean of the field."""
+        return self._mean
+
+    @mean.setter
+    def mean(self, mean):
+        self._mean = _set_mean_trend(mean, self.dim)
+
+    @property
+    def normalizer(self):
+        """:any:`Normalizer`: Normalizer of the field."""
+        return self._normalizer
+
+    @normalizer.setter
+    def normalizer(self, normalizer):
+        self._normalizer = _check_normalizer(normalizer)
+
+    @property
+    def trend(self):
+        """:class:`float` or :any:`callable`: The trend of the field."""
+        return self._trend
+
+    @trend.setter
+    def trend(self, trend):
+        self._trend = _set_mean_trend(trend, self.dim)
+
     @property
     def value_type(self):
         """:class:`str`: Type of the field values (scalar, vector)."""
         return self._value_type
 
     @value_type.setter
     def value_type(self, value_type):
-        """:class:`str`: Type of the field values (scalar, vector)."""
         if value_type not in VALUE_TYPES:
             raise ValueError("Field: value type not in {}".format(VALUE_TYPES))
         self._value_type = value_type
 
-    def __repr__(self):
-        """Return String representation."""
-        return "Field(model={0}, value_type={1})".format(
-            self.model, self.value_type
-        )
-
+    @property
+    def dim(self):
+        """:class:`int`: Dimension of the field."""
+        return self.model.field_dim
 
-def _names(name, cnt):
-    name = [name] if isinstance(name, str) else list(name)[:cnt]
-    if len(name) < cnt:
-        name += [name[-1] + str(i + 1) for i in range(cnt - len(name))]
-    return name
+    @property
+    def name(self):
+        """:class:`str`: The name of the class."""
+        return self.__class__.__name__
 
+    def _fmt_mean_norm_trend(self):
+        return fmt_mean_norm_trend(self)
```

Comment:
Why?
.
.
.
Okay, got it. Maybe add a small comment that this is a goodie for all the children.